scripts/parse_nagim.py

```diff
@@ -308,18 +639,20 @@ def transfer(
     if not tmp_dir:
         tmp_dir = tempfile.gettempdir()
 
+    namespace = 'main'
+
     if use_batch:
         hbatch = setup_batch(
             title='Transferring NAGIM data',
             keep_scratch=False,
-            tmp_bucket=f'cpg-{NAGIM_PROJ_ID}-{NAMESPACE}-tmp',
+            tmp_bucket=f'cpg-{NAGIM_PROJ_ID}-{namespace}-tmp',
             analysis_project_name=NAGIM_PROJ_ID,
         )
     else:
         hbatch = None
 
     for source in SOURCES_TO_PROCESS:
-        SOURCES[source].transfer(hbatch)
+        SOURCES[source].transfer_to_upload(hbatch, namespace)
 
     if use_batch:
         hbatch.run(wait=True, dry_run=dry_run)
```

Comment:
Just again, I think wait=True is a little dangerous when directly running the workflow.

Reply:
It's okay if the main process is interrupted, because all what the following code is doing is just the consistency checks. So if you kill the main process, you can manually watch the batch process finish, and then run the `parse` command that would start with those checks anyway